### PR TITLE
[gen-l10n] Cleans up formatting of the generated file

### PR DESCRIPTION
**Pull request by shihaohong**
Not sure how we'd add tests for this, but this cleans up the generated file so that the indentations are correct in the lookup method and also removes triple newlines wherever they occur in the file.

### CODE REVIEW

**Comment by shihaohong** [comment]
Maybe a way to test this is to check the generated output of a normal-ish output of the file for the presence of triple newlines and the right indentation? Any ideas @HansMuller

**Comment by shihaohong** [comment]
Wrote a test checking a few lines of the generated code. @HansMuller PTAL

**Comment by christopherfujino** [comment]
@shihaohong looks like this has a merge conflict.

File: packages/flutter_tools/lib/src/localizations/gen_l10n.dart [hunk 1]
```diff
@@ -354,6 +354,27 @@ String generateBaseClassMethod(Message message, LocaleInfo templateArbLocale) {
     .replaceAll('@(name)', message.resourceId);
 }
 
+// Add spaces to pad the start of each line. Skips the first line
+// assuming that its already present.
```
Comment (HansMuller) [inline packages/flutter_tools/lib/src/localizations/gen_l10n.dart:358]: that the padding is already present

File: packages/flutter_tools/lib/src/localizations/gen_l10n.dart [hunk 2]
```diff
@@ -354,6 +354,27 @@ String generateBaseClassMethod(Message message, LocaleInfo templateArbLocale) {
...
+String _addSpaces(String message, {int spaces}) {
+  bool isFirstLine = true;
+  return message
+    .split('\n')
+    .map((String value) {
+      if (isFirstLine) {
+        isFirstLine = false;
+        return value;
+      }
+      final StringBuffer newValue = StringBuffer();
```
Comment (HansMuller) [inline packages/flutter_tools/lib/src/localizations/gen_l10n.dart:368]: could use `return value.padLeft(spaces)`

**Review by HansMuller — APPROVED** [review]
LGTM

**Comment by jonahwilliams** [comment]
Sorry @shihaohong you need to rebase again 😅

**Pull request merged**